jenkins-config/managed-config-files/mavenSettings/dentos-settings/serverCredentialMappings.sandbox.yaml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
Move this to the template directory I specified above and symlink to that new file

Reply:
Done.. thank you